adminpages/paymentsettings.php

```diff
@@ -127,7 +127,7 @@
 					<label for="gateway"><?php _e('Payment Gateway', 'paid-memberships-pro' );?>:</label>
 				</th>
 				<td>
-					<select id="gateway" name="gateway" onchange="pmpro_changeGateway(jQuery(this).val());">
+					<select id="gateway" name="gateway" onchange="pmpro_changeGateway(jQuery('#gateway').val(), jQuery('#gateway_environment').val());">
```

Comment:
Why pass any arguments in at all if we can always just get the selects targeted by the ID?

Reply:
```suggestion
					<select id="gateway" name="gateway">
```